### DISCUSSION TITLE
Wyze Cam v2 - Flashing bootloader

### DISCUSSION THREAD

**Discussion by 87435** [openmiko/openmiko#123] [Q&A]
Hi all,  

First of all thank you for this project! It has given me hope after my cameras have become unreachable after using wz_mini_hacks. I am now able to use OpenMiko and access the camera with SSH (confirming that the hardware seems intact).  
 
[Per this section in the project's homepage](https://github.com/openmiko/openmiko#:~:text=If%20you%20flashed%20the%20custom%20HD%20Dafang%20bootloader%20you%20will%20need%20to%20revert%20to%20the%20original%20one.%20A%20copy%20of%20the%20old%20WyzeCam%20V2%20original%20bootloader%20is%20here%3A%20https%3A//github.com/openmiko/openmiko/blob/master/stock_firmware/wyzecam_v2/wyzecam_v2_stock_bootloader.bin) users need to restore their wyze v2 bootloader after flashing it with Dafang.   
1. What exactly is the procedure to do so?  
2. As ot today, should we still used the version linked ([here](https://github.com/openmiko/openmiko/blob/master/stock_firmware/wyzecam_v2/wyzecam_v2_stock_bootloader.bin) - uploaded 3 years ago) or is there a more recent version?

**Comment by bill-mcgonigle** [comment]
Does this help?

https://github.com/openmiko/openmiko/blob/00f72a6a2f701a038b39949fce239d54cd221065/initramfs_skeleton/init#L154

It looks like you create a file called `DO_BOOTLOADER_UPGRADE` in the root of the sdcard and then put the bootloader next to it renamed to `u-boot-lzo-with-spl.bin` .

Upon boot you should get a blue heartbeat light and on the sdcard the `DO_BOOTLOADER_UPGRADE` file will be deleted indicating a successful flash.

n.b. I haven't done this myself, just reading the script that gets compiled into the kernel.
